x/operator/types/codec.go

```diff
@@ -7,25 +7,12 @@ import (
 	"github.com/cosmos/cosmos-sdk/types/msgservice"
 )
 
-var (
-	amino = codec.NewLegacyAmino()
-
-	// ModuleCdc references the global erc20 module codec. Note, the codec should
-	// ONLY be used in certain instances of tests and for JSON encoding.
-	//
-	// The actual codec used for serialization should be provided to modules/erc20 and
-	// defined at the application level.
-	ModuleCdc = codec.NewProtoCodec(codectypes.NewInterfaceRegistry())
-
-	// AminoCdc is a amino codec created to support amino JSON compatible msgs.
-	AminoCdc = codec.NewAminoCodec(amino)
-)
-
-// NOTE: This is required for the GetSignBytes function
```

Comment:
With this removed, does sending transactions to the operator module still work?

Reply:
Yeah, Leon reminded me that the amino decode can be deprecated when developing a new project. I didn't encounter any problems after removing it. @leonz789